test/unit/php/includes/core/classes/class-test-export.php

```diff
@@ -40,6 +42,205 @@ public function test_setup_hooks(): void {
 		$this->assert_hooks( $hooks, $instance );
 	}
 
+	/**
+	 * Coverage for export.
+	 *
+	 * @covers ::export
+	 *
+	 * @return void
+	 */
+	public function test_export(): void {
+		$instance = Export::get_instance();
+
+		$this->assertFalse(
+			has_action( 'the_post', array( $instance, 'prepare' ) ),
+			'Failed to assert that the "the_post" action is not already added.'
+		);
+		$this->assertFalse(
+			has_filter( 'wxr_export_skip_postmeta', array( $instance, 'extend' ) ),
+			'Failed to assert that the "wxr_export_skip_postmeta" filter is not already added.'
+		);
+
+		$instance->export();
+
+		$this->assertSame(
+			10,
+			has_action( 'the_post', array( $instance, 'prepare' ) ),
+			'Failed to assert that the "the_post" action was added.'
+		);
+
+		$this->assertSame(
+			10,
+			has_filter( 'wxr_export_skip_postmeta', array( $instance, 'extend' ) ),
+			'Failed to assert that the "wxr_export_skip_postmeta" filter was added.'
+		);
```

Comment:
There's a helper to do this. Example [here](https://github.com/GatherPress/gatherpress/blob/develop/test/unit/php/includes/core/classes/class-test-block.php#L30).